coarse/api/aspect_purpose.py

```
@@ -0,0 +1,19 @@
+from fastapi import APIRouter
+from fastapi import UploadFile, File, HTTPException
```

Ruff (0.11.9)

`fastapi.UploadFile` imported but unused

Remove unused import

(F401)